server/src/main/java/org/apache/iotdb/db/service/DataNodeServerCommandLine.java

```diff
@@ -159,51 +128,52 @@ private void removeNodesFromCluster(String[] args)
             removeResp.getStatus().toString(), removeResp.getStatus().getCode());
       }
       logger.info(
-          "Submit remove datanode request successfully, "
+          "Submit remove-datanode request successfully, "
               + "more details are shown in the logs of confignode-leader and removed-datanode, "
-              + "and after the process of remove-datanode is over, "
+              + "and after the process of removing datanode is over, "
               + "you are supposed to delete directory and data of the removed-datanode manually");
     }
   }
 
   /**
-   * fetch all datanode info from ConfigNode, then compare with input 'ips'
+   * fetch all datanode info from ConfigNode, then compare with input 'ids'
    *
-   * @param endPorts data node ip:port, split with ','
+   * @param ids datanode id, split with ':'
    * @return TDataNodeLocation list
    */
-  private List<TDataNodeLocation> buildDataNodeLocations(String endPorts)
-      throws BadNodeUrlException {
+  private List<TDataNodeLocation> buildDataNodeLocations(String ids) {
     List<TDataNodeLocation> dataNodeLocations = new ArrayList<>();
-    if (endPorts == null || endPorts.trim().isEmpty()) {
+    if (ids == null || ids.trim().isEmpty()) {
       return dataNodeLocations;
     }
 
-    List<TEndPoint> endPoints = NodeUrlUtils.parseTEndPointUrls(endPorts);
+    // Currently support only single id, delete this line when supports multi ones
+    if (ids.split(":").length > 1) {
+      logger.info("Incorrect id format, usage: <node-id>");
```

Comment:
```suggestion
      logger.info("Incorrect id format, usage: -r <node-id>");
```

Reply:
I deem the '-r' somehow redundant, so I removed it in the command